tests/oteltest/LICENSE.txt

```diff
@@ -0,0 +1,13 @@
+# Copyright The OpenTelemetry Authors
```

Comment:
Other packages use the full license on a file named LICENSE

Reply:
Thanks -- should be fixed now.